reVX/utilities/reeds_cols.py

```diff
@@ -0,0 +1,274 @@
+
+# -*- coding: utf-8 -*-
+"""
+reVX ReEDS column addition utilities
+"""
+import json
+import geopandas as gpd
+import pandas as pd
+from warnings import warn
+
+from rex import Resource
+from reVX.utilities.utilities import to_geo, load_fips_to_state_map
+from reVX.version import __version__
+
+COUNTY_GDF_FP = ("https://www2.census.gov/geo/tiger/TIGER2021/COUNTY/"
+                 "tl_2021_us_county.zip")
+NREL_REGIONS = {
```

Comment:
you might end up getting a lot of these for various electrical system topologies (nemms regions, reeds ba's, etc...) You might consider putting this in a json in a folder of mappings. Assuming that somewhere we can just source a shape file too? There's a revx utility for doing the shape intersect but i'm not sure if you've included that here.

Reply:
Yea good point about the JSON file for the mapping. I will add that.

What is the reVX utility you are referring to? The GDF file is used to do a spatial overlay with the input supply curve to extract county FIPS values.

Reply:
Interesting. I did play with it a bit when I started out but for some reason moved away from it. I just tried it again and it does seem to do something similar to what I am doing for the county lookups so I adjusted the logic to lean on that class. Thanks for the tip!